client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -49,9 +51,9 @@ centroid varying vec2 varTexCoord;
 centroid varying float nightRatio;
 #endif
 varying highp vec3 eyeVec;
+varying float nightFactor;
 
-#ifdef ENABLE_DYNAMIC_SHADOWS
```

Comment:
I think you removed this line by error.

Reply:
Oh yes, I messed up testing the rebased code, hence this slipped through.